### PR TITLE
JWS-1973

### PR DESCRIPTION
**Pull request by rpelisse**
@gzaronikas This is my attempt to implement JWS-1973, however, I'm not really convinced of the added-value of handling webapps or drivers. All I did here is wrapped around get_url: (and copy:). I'm not sure it make sense to offer that to user. Most of them are just going to do it on their own.
Providing the template for DS configuration makes a bit more sense however.

WDYT?

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -59,3 +69,39 @@
         port: "6666"
       when:
         - modcluster_todo is defined
+
+    - set_fact:
+        app_name: info.war
+        app_url: 'https://drive.google.com/uc?export=download&id=1w9ss5okctnjUvRAxhPEPyC7DmbUwmbhb'
```
Comment (wolfc) [inline playbook.yml:75]: What's this?

Comment (rpelisse) [reply playbook.yml:75]: A silly [webapp](https://github.com/rpelisse/simple-servlet.git), I've build for the demos. It's just for testing purpose, but we can remove it (or using another available webapp to test). 

This webapp being stupid, it has the value of being very hard to fail to deploy :)

Comment (wolfc) [reply playbook.yml:75]: I don't like having references to "unknown" binary blobs.

Comment (rpelisse) [reply playbook.yml:75]: @wolfc as one should! To be honest, I was not expecting this PR to be accepted. But if we go with this, let's cleanup that part.
@gzaronikas do you have an easy/peasy app we could download to use for testing? Something less "unknown binary blob from Hell"?

Comment (bekionel) [reply playbook.yml:75]: @rpelisse Would the [sample webapp](https://tomcat.apache.org/tomcat-9.0-doc/appdev/sample/) from apache.org work for your purposes?

**Pull request closed without merging**